setup-express.js

```diff
@@ -0,0 +1,119 @@
+const expressWinston = require('express-winston');
```

Comment:
I'd move both this file and the `setup-winston.js` files into a `setup` folder.

Reply:
as soon as i understand how the hell imports work.